### ISSUE TITLE
Infinite loss showing after first few iterations of each epoch

### ISSUE THREAD

**Issue by shriramhr** [TheAILearner/A-CRNN-model-for-Text-Recognition-in-Keras#7]
Hi, 
I'm facing inf loss while training. Although val loss is constantly coming down, train loss always ends up as inf. I have tried lowering lr to 0.00001 also but it happens anyway. What val and train loss should we expect generally after its fully trained as per your dataset?
